boards/nxp/mimxrt595_evk/Kconfig.defconfig

```diff
@@ -27,10 +27,10 @@ config HEAP_MEM_POOL_ADD_SIZE_BOARD
 
 endif # DMA_MCUX_LPC
 
-# Turn on Device Level Power Management as we wish
+# Turn on system-managed device power management as we wish
 # to reconfigure the FlexSPI pins for power savings
 # when transitioning the SoC to Deep Low Power modes.
-config PM_DEVICE
+config CONFIG_PM_DEVICE_SYSTEM_MANAGED
```

Comment:
is this supposed to have CONFIG in the front, wouldnt this make it CONFIG_CONFIG_PM_DEVICE_SYSTEM_MANAGED